### ISSUE TITLE
Nested indent not working correctly

### ISSUE THREAD

**Issue by bumaociyuan** [Jintin/Swimat#6]
```
struct Error {

    enum ResultCode: Int {
        case fail = 0, success
    }

    let message : String
    let code : Int
}
```

`Swimat` ->

```
struct Error {

    enum ResultCode: Int {
    case fail = 0, success
    }

    let message : String
    let code : Int
}
```

**Comment by Jintin** [comment]
Thank you for your report
It's due to the switch case check will minus indent count and that also influence enum's situation.

```
switch {
case A:
    break
case B:
    break
}
```

You can reinstall and restart XCode to see if it is fixed in 1.2.4.

**Comment by bumaociyuan** [comment]
@Jintin thx fixed that
reinstall then it works well
